### DISCUSSION TITLE
Program Author

### DISCUSSION THREAD

**Discussion by not-nef** [marcelotduarte/cx_Freeze#1388] [Q&A]
Is it possible to specify a program author for msi? Because its always `UNKNOWN`!

**Comment by marcelotduarte** [comment]
You can use author or maintainer. See [source].
```
setup(
    name = "foo",
    version = "0.1",
    author = "My name",
    description = "My app!",
    options = {"build_exe": build_exe_options},
    executables = [Executable("foo.py", base=base)]
)
```

[source]: https://github.com/marcelotduarte/cx_Freeze/blob/main/cx_Freeze/windist.py#L935

**Reply by not-nef** [reply]
@marcelotduarte When i run the msi it gives me the following error:

`The installer has encountered an unexpected error installing this package. This may indicate a problem with this package. The error code is 2731.`

**Reply by marcelotduarte** [reply]
https://github.com/MicrosoftDocs/win32/blob/docs/desktop-src/Msi/windows-installer-error-messages.md

**Reply by not-nef** [reply]
Ok, so how i see this, this happens because the author parameter doesn’t exist. Am i doing something wrong?

**Reply by marcelotduarte** [reply]
There are some samples that you can check: https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze/samples
These samples work. Compare with your setup.
Without the author your aap works?

**Reply by not-nef** [reply]
> There are some samples that you can check: https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze/samples
> 
> These samples work. Compare with your setup.
> 
> Without the author your aap works?

Yes

**Reply by not-nef** [reply]
You can see the code of my setup [here](https://github.com/not-nef/Portal-Launcher/blob/main/setup.py).

When i add `author="nef"` to `setup()`, the msi doesnt work anymore
